prerendering.bs

```diff
@@ -658,6 +673,41 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+We need to ensure that the [=Accept-CH cache=], which is owned by user agent as a global storage, is not modified while prerendering, but is properly updated after activation. The following modifications ensure this:
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender-scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
+
+The [=prerendering browsing context/prerender-scoped Accept-CH cache=] is an [=ordered map=] of [=origins=] to [=client hints sets=].
```

Comment:
nit: of...to is slightly awkward, maybe `an [=ordered map=] from [=origin=] to [=client hint set=]`, or possibly the very verbose `an [=ordered map=] with [=map/keys=] which are [=origins=] and [=map/values=] which are [=client hint sets=].`

Reply:
Hmm, I was the one who suggested this plural "of ... to" phrasing, and I think it matches existing spec verbiage. E.g. https://html.spec.whatwg.org/#concept-worklet-module-responses-map or https://wicg.github.io/navigation-api/#navigation-upcoming-traverse-navigations .

Reply:
There is the pattern of "map of typeA to typeB" in my corpus.. I cannot tell if it is widely used, but it seems understandable. 
( I find some sentences in this specifiation also use this pattern, so can I keep it for now and update them if we find "map from typeA to typeB" is preferred to  "map of typeA to typeB" ?